frame/bounties/src/lib.rs

```diff
@@ -660,12 +785,685 @@ decl_module! {
+					// Ensure Subbounty is in active state
+					if let SubBountyStatus::Active { ref subcurator } =
+						subbounty.status
+					{
+						ensure!(
+							signer == *subcurator,
+							Error::<T>::RequireSubCurator,
+						);
+						// Move the subbounty state to Pending payout.
+						subbounty.status = SubBountyStatus::PendingPayout {
+							subcurator: signer,
+							beneficiary: beneficiary.clone(),
+							unlock_at: system::Pallet::<T>::block_number() +
+								T::BountyDepositPayoutDelay::get(),
+						};
+						Ok(())
+					} else {
+						Err(Error::<T>::UnexpectedStatus.into())
+					}
+				}
+			)?;
+			// Trigger the event SubBountyAwarded
+			Self::deposit_event(
+				Event::<T>::SubBountyAwarded(
+					bounty_id,
+					subbounty_id,
+					beneficiary
+				)
+			);
+		}
+
+		/// Claim the payout from an awarded subbounty after payout delay.
+		///
+		/// The dispatch origin for this call may be any signed origin.
+		///
+		/// Call works independent of parent bounty state,
+		/// No need for parent bounty to be in active state.
+		///
+		/// The Beneficiary is paid out with agreed bounty value.
+		/// SubCurator fee is paid & curator deposit is unreserved.
+		///
+		/// Subbounty must be in "PendingPayout" state, for
+		/// processing the call. And instance of subbounty is
+		/// deallocated from DB on successful call completion.
+		///
+		/// - `bounty_id`: ID pair Bounty ID.
+		/// - `subbounty_id`: ID pair SubBounty ID to cancel.
+		#[weight = <T as Config>::WeightInfo::claim_subbounty()]
+		fn claim_subbounty(origin,
+			#[compact] bounty_id: BountyIndex,
+			#[compact] subbounty_id: BountyIndex,
+		) {
+			let _ = ensure_signed(origin)?;
+
+			// Ensure subbounty is in expected state
+			SubBounties::<T>::try_mutate_exists(
+				bounty_id,
+				subbounty_id,
+				|maybe_subbounty| -> DispatchResult {
+					let subbounty = maybe_subbounty
+						.as_mut()
+						.ok_or(Error::<T>::InvalidIndex)?;
+
+					if let SubBountyStatus::PendingPayout {
+						ref subcurator, ref beneficiary, ref unlock_at
+					} = subbounty.status {
+						// Ensure block number is elapsed for
+						// processing the claim.
+						ensure!(
+							system::Pallet::<T>::block_number() >= *unlock_at,
+							Error::<T>::Premature,
+						);
+
+						// Make curator fee payment
+						let subbounty_account = Self::bounty_account_id(subbounty_id);
+						let balance = T::Currency::free_balance(&subbounty_account);
+						let fee = subbounty.fee.min(balance); // just to be safe
+						let payout = balance.saturating_sub(fee);
+
+						// unreserve the subcurator deposit
+						let _ = T::Currency::unreserve(
+							&subcurator,
+							subbounty.curator_deposit,
+						); // should not fail
+
+						// Make payout to subcurator
+						let _ = T::Currency::transfer(
+							&subbounty_account,
+							&subcurator,
+							fee,
+							AllowDeath,
+						); // should not fail
+
+						// Make payout to beneficiary
+						let _ = T::Currency::transfer(
```

Comment:
same, you should do it for all applicable `let _ = <some_result>` in the pallet.